_episodes/05-history.md

```diff
@@ -363,6 +363,26 @@ moving backward and forward in time becomes much easier.
 > 4. `$ git checkout <unique ID of last commit> data_cruncher.py`
 >
 > 5. Both 2 and 4
+>
+>
+> > ## Solution
+> >
+> > The answer is (5)-Both 2 and 4. 
+> > 
+> > The checkout command restores files from the repository, overwriting the files in your working 
```

Comment:
Maybe quote `checkout` ?

Reply:
Good catch. Will you submit a PR?

Reply:
https://github.com/swcarpentry/git-novice/pull/594

Reply:
Thanks for the contribution and quick response!